features/helper_methods/let.feature

```diff
@@ -7,6 +7,9 @@ Feature: let and let!
   the method it defines is invoked. You can use `let!` to force the method's
   invocation before each example.
 
+  By default, `let` is threadsafe, but you can configure it not to be
+  by disabling `config.threadsafe`, which makes `let` perform a bit faster.
+
```

Comment:
Works for me. Mostly felt the need for the cuke, b/c none of the unit-level tests ensured that the CLI option integrated with the config option. With config option removed, a simple note is much easier to maintain :)